changelog/fragments/fix-trim-format-dns1123-functions.yaml

```
@@ -0,0 +1,11 @@
+entries:
+  - description: >
+      TrimDNS1123Label changed to return VALID DNS1123 strings. It
+      previously blindly trimmed them potentially creating invalid labels.
+
+      FormatOperatorNameDNS1123 would return strings that were invalid if
+      they begin or end with non-alphanumeric or hyphens.
```

These are internal functions. This should read something like
```suggestion
      Fixed invalid object names generated for long package names passed to `run packagemanifests`.
```